src/app/api/test/seed-photos/route.ts

```
@@ -0,0 +1,29 @@
+  const existing = await albumRepo.listAlbums(userId, 1, 0)
+  const album = existing.albums[0] || await albumRepo.createAlbum(userId, { name: 'Demo Album', description: 'Seeded photos' })
+  // Add photos
+  let order = 0
+  for (const url of photos) {
+    await photoRepo.createPhoto(userId, { url, caption: 'Demo', albumId: album.id, order, isPrimary: order === 0 })
+    order++
+  }
+  return new Response(JSON.stringify({ success: true, data: { albumId: album.id } }), { status: 201, headers: { 'Content-Type': 'application/json' } })
+}
+
+export const POST_Handler = requireOnboarding(POST)
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)